### ISSUE TITLE
Improvements to Project List in UI

### ISSUE THREAD

**Issue by tnelson-doghouse** [amazeeio/lagoon#2346]
After discussion at Doghouse, the suggestion was that, as the project list grows, it'll become more difficult to find things.  Suggestions included:

-     Represent projects as cards so that more can fit onto the screen at a time
-     Represent projects as a table (presumably in a smaller font) so that more fit onto the screen at a time

Obviously the idea was that these were alternative settings, not a replacement for the existing setup.
Not the most pressing problem at the moment, but I thought we should relay it to you guys as feedback.

**Comment by dan2k3k4** [comment]
Tested 2-column layout:
![image](https://user-images.githubusercontent.com/158704/104644459-1580e880-56ae-11eb-9a7e-468c979132d5.png)

I personally find it easier to glance in the same "starting area" for a project name, instead of checking left and right all the time. Especially if you add new projects that show up somewhere in the middle as we sort the projects list be alphabetical order.

I think a card layout would only work if we had images and perhaps sorted by "project creation date", as that might make it easier to notice the project / find it at a glance, rather than trying to read left-right for each row.

**Comment by tnelson-doghouse** [comment]
I'm realising my comment about a table wasn't clear.  I meant something with multiple columns that had one column for project name, and other columns for project attributes, but and the writing and spacing were smaller than the current setup.  That's why I suggested the cards idea as an alternative layout idea, with the thought that cards would have a larger font and more spacing (much like what you have above).  

HTH,

**Comment by dan2k3k4** [comment]
Ah OK, perhaps something like this @tnelson-doghouse?

![table-projects-output](https://user-images.githubusercontent.com/158704/104972204-556b0700-59f1-11eb-96f0-d4f80d0ad2e3.png)

Although I don't show any of the `<customer>` data as thats currently empty:
https://github.com/amazeeio/lagoon/blob/836abd531151b9f7d61cf0ab425816cd4b237407/services/ui/src/components/Projects/index.js#L100-L102

I believe @justinlevi may have an idea on what data will be displayed there.

**Comment by dan2k3k4** [comment]
Another thought would be support both a card view and a table view, similar to: https://codepen.io/aybukeceylan/full/OJRNbZp

![table-card](https://user-images.githubusercontent.com/158704/104972857-42593680-59f3-11eb-906e-9889ff86b0a1.gif)

**Comment by tnelson-doghouse** [comment]
Both of those look good.  My observation has been that there are two types of UI users, those who want simple, and those who want info.  I'm definitely in the "info" category.  

Within the "info" category, there are two useful options:

- Display lots of info about each thing
- Show as many things as possible on the screen at once (while still hopefully showing multiple information about each item); this is handy for locating the thing you want quickly

From my POV, your suggestions are moving towards both of these last two options.  It may be that the users who want Simple will not be using Lagoon, but I thought I should mention them in case I'm wrong (they probably like the current interface, with big text and just the most relevant information).  

So, in short, definitely happy with where this is going.  

Thanks!

**Comment by brett-sargeant** [comment]
being able to order the list would be extremely useful.

**Comment by tobybellwood** [comment]
We'll be making changes to the projects list in UIv2 :smile:
